sirepo/package_data/static/js/sirepo.js

```diff
@@ -3941,6 +3949,7 @@ SIREPO.app.controller('SimulationsController', function (appState, cookieService
         appState.saveQuietly('simFolder');
     });
     loadList();
+    wakeAgent();
```

Comment:
As @moellep said, let's avoid roundtrips, and this isn't the correct place anyway.